src/mainsite/components/PriceModel.tsx

```diff
@@ -78,6 +78,8 @@ const Marker: FC<MarkerProps> = ({ alt, icon, peRatio, symbol }) => {
             ? undefined
             : symbol === "DIS"
             ? "https://www.google.com/finance/quote/DIS:NYSE"
+            : symbol === "ETH
```

Comment:
Looks like there is a terminating quote `"` missing here.

Reply:
ups, sorry, fixing it